packages/frontend/src/components/dialogs/CreateChat/index.tsx

```diff
@@ -663,8 +663,25 @@ export function CreateGroup(props: CreateGroupProps) {
     })
   }
 
+  const submitForm = (ev: React.FormEvent) => {
+    ev.preventDefault()
+    if (groupName === '') {
+      setErrorMissingGroupName(true)
+      return
+    }
+    finishCreateGroup()
+      .then(groupId => {
+        if (groupId) {
+          selectChat(accountId, groupId)
+        } else {
+          // TODO: handle error
+        }
+      })
+      .finally(onClose)
+  }
+
   return (
-    <>
+    <form onSubmit={submitForm}>
```

Comment:
This causes double scroll

<img width="406" height="475" alt="Image" src="https://github.com/user-attachments/assets/f91ddb8b-1f8f-495e-aead-6e02cf21a4b3" />

Technically this is also the case for "create channel" but there it only happens on super short windows sizes, currently.